src/bionics.cpp

```diff
@@ -2096,13 +2150,23 @@ int bionic::get_quality( const quality_id &quality ) const
     return item( i.fake_item ).get_quality( quality );
 }
 
+bool bionic::is_muscle_powered() const
+{
+    for( const itype_id fuel : this->info().fuel_opts ) {
```

Comment:
```suggestion
    for( const itype_id &fuel : info().fuel_opts ) {
```
But better would be using `std::find`, or maybe change `fuel_opts` into a set and query it directly.

Reply:
I used std::find, I'm not too sure how to handle making fuel_opts a set